### PR TITLE
[VS Code] Update document

### PR DESCRIPTION
**Pull request by yiyione**

### CODE REVIEW

File: contrib/pai_vscode/README.md [hunk 1]
```diff
@@ -48,7 +49,7 @@ After added a cluster configuration, you can find the cluster in *PAI CLUSTER EX
 Submit V1 job:
```
Comment (debuggy) [inline contrib/pai_vscode/README.md]: there should be a blank line or a ```-``` if you want line break

File: contrib/pai_vscode/README.md [hunk 2]
```diff
@@ -48,7 +49,7 @@ After added a cluster configuration, you can find the cluster in *PAI CLUSTER EX
...
 You can create a job configuration and submit to OpenPAI as below steps.
 
-1. Double click *Create Job Config...* in OpenPAI cluster Explorer, and then specify file name and location to create a job configuration file.
+1. Make sure the value of 'protocol_version' property in cluster configuration is '1'. Double click *Create Job Config...* in OpenPAI cluster Explorer, and then specify file name and location to create a job configuration file.
```
Comment (debuggy) [inline contrib/pai_vscode/README.md]: maybe it's better to use ``` in places where need quote

File: contrib/pai_vscode/README.md [hunk 3]
```diff
@@ -123,7 +124,7 @@ There are two parts in the side bar.
 | Name                            | Description                               |
 | ------------------------------- | ----------------------------------------- |
 | PAI: Add PAI Cluster            | Add a new OpenPAI cluster                 |
-| PAI: Open Dashboard             | View OpenPAI cluster in browser           |
+| PAI: Open Website               | View OpenPAI cluster in browser           |
```
Comment (debuggy) [inline contrib/pai_vscode/README.md]: why in my extension the command is still open dashboard

File: contrib/pai_vscode/README.md (hunk 1 above)
Comment (yiyione) [reply contrib/pai_vscode/README.md]: Done.

File: contrib/pai_vscode/README.md (hunk 2 above)
Comment (yiyione) [reply contrib/pai_vscode/README.md]: Done.

**Pull request merged**